src/System.Drawing.Primitives/pkg/System.Drawing.Primitives.pkgproj

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Drawing.Primitives.csproj">
+    </ProjectReference>
```

Comment:
Added and verified locally. The ref for net45 is using the facade itself. Is that expected? I tried to add net45 to the ref csproj, but i get errors. I see system.data.common and system.data.sqlclient also has this same structure, using the desktop facade as ref assembly.